code/addons/docs/src/blocks/components/Toolbar.tsx

```diff
@@ -97,5 +97,26 @@ export const Toolbar: FC<ToolbarProps> = ({
         </>
       )}
     </Wrapper>
+    {isLoading ? (
+      <Wrapper key="right">
+        <IconPlaceholder />
+      </Wrapper>
+    ) : (
+      baseUrl &&
+      storyId && (
+        <Wrapper key="right">
+          <IconButton key="opener" asChild>
+            <a
+              href={getStoryHref(baseUrl, storyId)}
+              target="_blank"
+              rel="noopener noreferrer"
+              title="Open canvas in new tab"
```

Comment:
Let's use an `aria-label` here instead. I have a branch that replaces `IconButton` with a button that provides aria labels and tooltips with the same prop, so I'll take care of migrating your code after merging, but let's err on the side of accessibility until then!

```suggestion
              aria-label="Open canvas in new tab"
```